docs/samples/custom/torchserve/docs/canary.md

```diff
@@ -21,12 +22,13 @@ Change the path and deploy
 apiVersion: "serving.kubeflow.org/v1beta1"
 kind: "InferenceService"
 metadata:
- name: "torchserve-custom"
+  name: "torchserve-custom"
 spec:
- predictor:
-   canaryTrafficPercent: 20
-   pytorch:
-     storageUri: "pvc://model-store-claim-1"
+  predictor:
+    canaryTrafficPercent: 20
+    containers:
+    - image: {username}/torchserve_v2:latest
```

Comment:
how do you create the `torchserve_v2` image ?

Reply:
The same steps we use for creating the other image.